tests/sentry/integrations/slack/service/test_slack_service.py

```diff
@@ -40,6 +49,114 @@ def test_simple(self) -> None:
         assert result == "admin@localhost archived BAR-1"
 
 
+@mock.patch("sentry.integrations.slack.service")
+class TestNotifyAllThreadsForActivity(TestCase):
+    def setUp(self) -> None:
+        self.service = SlackService.default()
+        self.activity = Activity.objects.create(
+            group=self.group,
+            project=self.project,
+            type=ActivityType.SET_IGNORED.value,
+            user_id=self.user.id,
+            data={"ignoreUntilEscalating": True},
+        )
+
+        self.rule_action_uuid = str(uuid4())
+        self.notify_issue_owners_action = [
+            {
+                "targetType": "IssueOwners",
+                "fallthroughType": "ActiveMembers",
+                "id": "sentry.mail.actions.NotifyEmailAction",
+                "targetIdentifier": "",
+                "uuid": self.rule_action_uuid,
+            }
+        ]
+        self.rule = self.create_project_rule(
+            project=self.project, action_match=self.notify_issue_owners_action
+        )
+        self.rule_fire_history = RuleFireHistory.objects.create(
+            project=self.project,
+            rule=self.rule,
+            group=self.group,
+            event_id=456,
+            notification_uuid=str(uuid4()),
+        )
+        self.parent_notification = NotificationMessage.objects.create(
+            rule_fire_history_id=self.rule_fire_history.id,
+            rule_action_uuid=self.rule_action_uuid,
+        )
+        with assume_test_silo_mode(SiloMode.CONTROL):
+            self.integration = self.create_integration(
+                organization=self.organization,
+                name="slack",
+                provider="slack",
+                external_id="slack:1",
+                metadata={"access_token": "xoxb-access-token"},
+            )
+
+    def test_none_group(self, mock_logger):
+        self.activity.update(group=None)
+        self.service.notify_all_threads_for_activity(activity=self.activity)
+        assert mock_logger.info.called_with("no group associated on the activity, nothing to do")
```

Comment:
this assertion is doing nothing -- you either want `mock_logger.info.assert_called_with(...)` or to assert on `call_args_list`

~generally you want to make sure your test fails _before_ the implementation such that you don't end up with tests which aren't actually testing anything

additionally, `caplog` is usually easier to use than mocking a logger -- and is built into pytest!